dashboard/test/factories/curriculum_factories.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: reference_guides
```

Comment:
Can you remove this table information from the top of this file since we are going to use this for all curriculum factories? 

Also should we add an item to the backlog to move the rest of the curriculum factories here?